en/controllers/request-response.rst

```diff
@@ -1067,6 +1067,26 @@ criteria are met:
     is very application specific. We recommend you build your own ``CORSMiddleware``
     if you need one and adjust the response object as desired.
 
+Running logic after the Response has been sent
+----------------------------------------------
+
+In fastcgi based environments you can listen to the ``Server.terminate`` event
+to run logic **after** the response has been sent to the client. Make sure your
+application's **webroot/index.php** contains the following::
+
+    $server = new Server(new Application(dirname(__DIR__) . '/config'));
+
+    $request = ServerRequest::fromGlobals();
+    $response = $server->run($request);
+    $server->emit($response);
+    $server->terminate($request, $response);
+
+.. warning::
+   In non fastcgi environments the ``Server.terminate`` event is fired before
+   the response is sent.
```

Comment:
Need to add a warning about the scope of the request object used unless we want to make the modification suggested here: https://github.com/cakephp/cakephp/pull/17514#issuecomment-1872567385

Reply:
We can just update the example to pull from the container:
https://github.com/cakephp/cakephp/pull/17514#issuecomment-1888374363